.circleci/config.yml

```diff
@@ -585,14 +609,16 @@ workflows:
             - cli-extra-test
             - tbl2tbl-itest
             - bigquery-itest
-            - redshift-itest
-            - redshift-itest-old-pandas
-            - redshift-itest-no-pandas
+            - redshift-itest-no-s3
+            - redshift-itest-s3
+            - redshift-itest-s3-old-pandas
+            - redshift-itest-s3-no-pandas
             - postgres-itest
             - mysql-itest
             - cli-1-itest
             - cli-2-itest
             - cli-3-itest
+            # TODO: Decide on name ordering
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker